### PR TITLE
restructure folders, rename templates to match module, update grunt

### LINKED ISSUE #26: Store templates with each module instead of separately
Having template files separate to the js src is confusing, especially when looking around this module's files in a file browser or terminal.

Also it would be nice to have `templates/aif/select-input.html` instead of `templates/aif/select-input/template.html`.

Before

```
angular-input-fields/
└── src
    ├── scripts
    │   ├── init.js
    │   ├── select-input
    │   │   ├── directives.js
    │   │   └── templates.js
    │   └── text-input
    │       ├── directives.js
    │       └── templates.js
    └── templates
        └── aif
            ├── select-input
            │   └── template.html
            └── text-input
                └── template.html
```

after

```
angular-input-fields/
└── src
    ├── angular-input-fields.js
    └── aif-text-input
        ├── directives.js
        ├── templates.js
        └── templates
            └── aif
                ├── select-input.html
                └── text-input.html
```

This would work nicely with #17

### PR DESCRIPTION
**Pull request by hippogriffic**
@incuna/frontend please review

fixes #26

### CODE REVIEW

**Comment by pandalion** [comment]
Wow, nice grunt and angular stuff :D It looks good to me, not sure if anybody else should look before merging...

File: src/text-input/templates/aif/text-input.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="text-input-wrapper">
+    <input
+        class="text-input"
+        id="{{ fieldId }}"
+        type="text"
+        ng-model="model"
+        ng-disabled="disable"
+        >
```
Comment (wytrych) [inline src/text-input/templates/aif/text-input.html]: Not really sure that having the `text-input` gives us anything but you might as well keep it

**Comment by wytrych** [comment]
Looks good

File: src/select-input/templates/aif/select-input.html [hunk 2]
```diff
@@ -3,9 +3,11 @@
         class="select"
         ng-model="model"
         ng-disabled="disable"
-        ng-options="choice.value as choice.display_name for choice in choices"
         id="{{ fieldId }}"
         name="{{ fieldId }}"
+        ng-focus="form.fieldId.hasFocus = true"
+        ng-blur="form.fieldId.hasFocus = false"
```
Comment (hippogriffic) [inline src/select-input/templates/aif/select-input.html]: this stuff needs to be removed

File: src/text-input/templates/aif/text-input.html (hunk 1 above)
Comment (hippogriffic) [inline src/text-input/templates/aif/text-input.html]: yeah I ended up not using it, or actually this library in the project I'm working on but oh well :)

**Comment by pandalion** [comment]
@hippogriffic happy for you to merge if you're done :)

**Pull request merged**